### ISSUE TITLE
Transition Into Down Tilt 2 Motion Via Status Script

### ISSUE THREAD

**Issue by FatherOfEgg** [ZeDespo/Ultimate-Ganondorf#15]
We can change into the down tilt 2 motion using a status script, and therefore do not need to do [this check](https://github.com/ZeDespo/Ultimate-Ganondorf/blob/51cc07784103fa20c267dad6498784a8e2ecb603/src/ganon/frame.rs#L26) every frame.

**Comment by FatherOfEgg** [comment]
@ZeDespo Do you still want the user to [hold down the left stick](https://github.com/ZeDespo/Ultimate-Ganondorf/blob/51cc07784103fa20c267dad6498784a8e2ecb603/src/ganon/down_tilt_followup.rs#L11), alongside the other criteria, in order to go into down tilt 2? Other moves that have 2 or more parts to them usually only care about if you pressed attack again.

**Comment by ZeDespo** [comment]
Yeah I definitely think the stick should be held down. My logic is that someone can cancel down tilt into another move aside from down tilt 2; however, with how the move is, I don't think it's cancellable into anything good. 

If you disagree, I'd be more than happy to be proven wrong.
